docs/application/dotnet/samples/system/overview.md

```diff
@@ -116,14 +116,14 @@ for TD:
 		<tr>
 			<td><img alt="" height="180" src="media/w79pedometer.png" width="180"/></td>
 			<td>
-			<p><strong>(W) Pedometer</strong> [In progress]</p>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/Pedometer" target="_blank"><strong>(W) Pedometer</strong></a></p>
 			<p>This sample application demonstrates how to obtain the pedometer sensor data using <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.Sensor.Pedometer.html" target="_blank">Tizen.Sensor.Pedometer</a>.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="180" src="media/w80squatcounter.png" width="180"/></td>
 			<td>
-			<p><strong>(W) SquatCounter</strong> [In progress]</p>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/SquatCounter" target="_blank"><strong>(W) SquatCounter</strong></a></p>
```

Comment:
Side issue : The title of the READMD.md in this link is SpeechToText, not SquatCounter.

Reply:
Thanks, Fixed!